packages/python-packages/apiview-copilot/evals/_runner.py

```diff
@@ -8,47 +8,35 @@
 
 from azure.ai.evaluation import evaluate
 from azure.identity import AzurePipelinesCredential
-from evals._custom import CustomAPIViewEvaluator
-from src._apiview_reviewer import ApiViewReview
+from evals._custom import EVALUATORS
 from src._settings import SettingsManager
 from tabulate import tabulate
 
 DEFAULT_NUM_RUNS: int = 1
 
-
-def _review_apiview(query: str, language: str):
-    """Internal global callable for evals framework."""
-    reviewer = ApiViewReview(target=query, language=language, base=None)
-    review = reviewer.run()
-    reviewer.close()
-    return {"actual": review.model_dump_json()}
-
-
 class EvalRunner:
     """Class to run evals for APIView copilot."""
 
-    def __init__(self, *, language: str, test_path: str, num_runs: int = DEFAULT_NUM_RUNS):
+    def __init__(self, *, language: str, test_path: str, evaluator_type: str = "apiview", num_runs: int = DEFAULT_NUM_RUNS):
         self.language = language
         self.test_path = test_path
+        self.evaluator_type = evaluator_type
         self.num_runs = num_runs
         self.settings = SettingsManager()
 
+        if evaluator_type not in EVALUATORS:
+            raise ValueError(f"Unknown evaluator type: {evaluator_type}. Available: {list(EVALUATORS.keys())}")
+
         self._tests_directory = pathlib.Path(__file__).parent / "tests" / self.language
         self._test_file = pathlib.Path(test_path).name
-        self._weights: dict[str, float] = {
-            "exact_match_weight": 0.7,  # Exact match (rule id and line number)
-            "groundedness_weight": 0.2,  # Staying grounded in guidelines
-            "similarity_weight": 0.1,  # Similarity between expected and actual
-            "false_positive_penalty": 0.3,  # Penalty for false positives
-            "fuzzy_match_bonus": 0.2,  # Bonus for fuzzy match (right rule, wrong line)
-        }
 
     def run(self):
         """Run the evaluation."""
-        custom_eval = CustomAPIViewEvaluator()
+        evaluator_class = EVALUATORS[self.evaluator_type]
+        custom_eval = evaluator_class()
```

Comment:
Two things.
1) refactor `custom_eval` to maybe just `evaluator`.
2) we should have some way so that we don't need to specify `evaluator_type`. Perhaps that is something that can be stored in the JSONL?  Or, we could create something like a YAML format that links the JSONL file to its proper class. Let's discuss at our sync today.